src/middlewares/validators/orders/createOrderPayload.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi');
+
+const createOrderPayloadValidator = (req, res, next) => {
```

Comment:
We need to add more info to order: first name, middle name, surname of user. Also address, phone number.

Reply:
Separate

Reply:
Fixed